mobile_app/lib/view/cafe_detail/cafe_detail_footer.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:mobile_app/constants/color.dart';
+import 'package:mobile_app/view/common/button/button_size.dart';
+import 'package:mobile_app/view/common/button/custom_icon_button.dart';
+import 'package:mobile_app/util/common.dart';
+import 'package:mobile_app/view/common/button/custom_button.dart';
+
+class CafeDetailFooter extends StatelessWidget {
+  final String cafeId;
+  final Function()? onMenuScroll;
+  final Function()? onMapScroll;
+
+  CafeDetailFooter({required this.cafeId, required this.onMenuScroll, required this.onMapScroll});
+
+  @override
+  Widget build(BuildContext context){
+    return Container(
+      width: MediaQuery.of(context).size.width,
+      decoration: BoxDecoration(color: Palette.whiteTransparentBG),
+      child: Row(
+        children: [
+          if(onMenuScroll != null)
+            CustomIconButton(
+              size: iconButtonSize,
+              content: IconButtonContent(text: '메뉴', icon: Icons.coffee_rounded),
+              onPressed: onMenuScroll!,
+            ),
+          if(onMapScroll != null)
+            CustomIconButton(
+              size: iconButtonSize,
+              content: IconButtonContent(text: '지도', icon: Icons.map_outlined),
+              onPressed: onMapScroll!,
+            ),
+         Expanded(
+           child: CustomButton(
+             size: buttonSize,
+             child: Text('카페 공유하기'),
+             onPressed: () => handleLinkShareClick(
+                 'https://www.coffeehmm.com/cafe/$cafeId'),
+           ),
+         )
+        ],
+      )
+    );
+  }
+}
+
+final iconButtonSize = ButtonSize(
+    height: 72,
+    padding: EdgeInsets.only(left: 10, right: 10, top: 12, bottom: 0)
+);
+final buttonSize = ButtonSize(
+    height: 72,
+    padding: EdgeInsets.only(left: 20, right: 20, top: 12, bottom: 20)
+);
```

Comment:
```suggestion
final _iconButtonSize = ButtonSize(
    height: 72,
    padding: EdgeInsets.only(left: 10, right: 10, top: 12, bottom: 0)
);
final _buttonSize = ButtonSize(
    height: 72,
    padding: EdgeInsets.only(left: 20, right: 20, top: 12, bottom: 20)
);
```

파일 내에서만 사용되는 변수는 `_` prefix를 붙여야 export되지 않습니다.
다른 파일에서 공유될 가능성이 있다면 이 상수는 `button_size.dart`쪽에 정의하는 것이 더 맞아 보이네요